storybook/src/stories/Form/TextAreaField.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+import {  text, select } from "@storybook/addon-knobs";
+import { TextAreaField } from 'scorer-ui-kit';
+
+const Container = styled.div`
+  margin: 20px;
+`;
+
+export default {
+  title: 'Form/molecules',
+  component: TextAreaField,
+  decorators: []
+};
+
+
+export const TextareaField = () => {
```

Comment:
When the Storybook function name clashes with the UI Kit's component name you can use the same thing and just prefix is with `_`. For example, `_TextAreaField`.

This works though and doesn't have any effect elsewhere so it doesn't need changing, just a tip I thought I would share! :)

Reply:
Thanks for the tip, updated! 😄